tests/integration/targets/terraform_provider/terraform.tfstateshow

```diff
@@ -113,6 +113,28 @@
           },
           "sensitive_values": {}
         }
+      ],
```

Comment:
Can you also update the ansible_provider.tf with the child module that produces this output?

Reply:
Sounds good, I will add that later today!

Reply:
I added the module in the path `./modules/example` and slightly changed what the tfstate has to make it a simpler demo

In the process of this I changed the source to the public one so that `terraform init` would work. The new tfstateshow is the output of `terraform show -o json` and putting it through DuckDuckGo's JSON formatter

Reply:
Please let me know if I should revert the change to the workaround one for now